### PR TITLE
feat: support CentOS image for dashboard

### PR DESCRIPTION
**Pull request by bzp2010**

### CODE REVIEW

File: dashboard/Dockerfile.centos [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+FROM centos:8 as prod
+
+WORKDIR /usr/local/apisix-dashboard
+
+COPY --from=api-builder /usr/local/apisix-dashboard/output/ ./
+
+COPY --from=fe-builder /usr/local/apisix-dashboard/output/ ./
+
+RUN mkdir logs
```
Comment (tao12345666333) [inline dashboard/Dockerfile.centos:66]: Is this necessary?

**Review by tao12345666333 — COMMENTED** [review]
Please check the failed job.

File: dashboard/Dockerfile.centos (hunk 1 above)
Comment (bzp2010) [reply dashboard/Dockerfile.centos:66]: Hi, @tao12345666333. Do you mean providing CentOS images?

Comment (bzp2010) [reply dashboard/Dockerfile.centos:66]: If this is the question, as there is some user feedback that using alpine as the base image causes some strange DNS issues and would like to add centos system support.

**Review by nic-chen — APPROVED** [review]

File: dashboard/Dockerfile.centos (hunk 1 above)
Comment (bzp2010) [reply dashboard/Dockerfile.centos:66]: If you are referring to the question of why mkdir is needed, this is copied from the original code, and if we don't create it beforehand, we might run into permission problems in some environments.

**Review by shuaijinchao — APPROVED** [review]

File: dashboard/Dockerfile.centos (hunk 1 above)
Comment (tao12345666333) [reply dashboard/Dockerfile.centos:66]: ok

**Pull request merged**